docs/projects/tad/adrs/0005-ai-verify-technical-tests.md

```diff
@@ -0,0 +1,40 @@
+# TAD-0005 Use AI Verify for technical tests
+
+## Context
+
+The AI Verify project is set up in a modular way, and the technical tests is one of the modules. The AI Verify
+team is developing a feature which makes it possible to run the technical tests using an API, by using
+a python file calling a method to run a test and providing the required configuration, like which
+model and dataset to use and some test specific configuration. The result of the test are returned in a JSON format,
+which can be processed as we want, like write it to a file or store it in a database.
+
+### Pros
+
+* We have several technical tests we can use of the shelf.
+* Tests are set up in a generic way using interfaces which allows others, like ourselves, to create their own plugins.
+* Loading models, pipelines and data is done through the AI Verify toolkit, which does validation
+ and support checks.
+
+### Cons
+
+* The result of a test is dependent on how AI Verify has implemented it.
```

Comment:
perhaps we can be more specific here and name that we have less flexibility in what is logged into the process report and what is written into the result report.